valhalla/baldr/graphreader.h

```diff
@@ -637,16 +637,19 @@ class GraphReader {
   /**
    * Method to get the begin node of an edge by using its opposing edges end node
    * @param edge    the edge whose begin node you want
-   * @param tile    reference to a pointer to a const tile
+   * @param tile    reference to a pointer to a const tile containing the begin node
    * @return        returns GraphId of begin node of the edge (empty if couldn't find)
    */
   GraphId GetBeginNodeId(const DirectedEdge* edge, graph_tile_ptr& tile) {
-    // grab the node
-    if (!GetGraphTile(edge->endnode(), tile))
+    // grab the end node maybe in an adjacent tile
+    graph_tile_ptr maybe_other_tile = tile;
+    if (!GetGraphTile(edge->endnode(), maybe_other_tile))
       return {};
-    const auto* node = tile->node(edge->endnode());
-    // grab the opp edges end node
-    const auto* opp_edge = tile->directededge(node->edge_index() + edge->opp_index());
+    const auto* node = maybe_other_tile->node(edge->endnode());
+    // grab the opp edge also could be in this adjacent tile
```

Comment:
I was wondering if we could write out "opposite" in the comment? Also maybe some punctuation is missing in the sentence? :thinking:

Reply:
maybe a comma missing, but "opp edge" should be readily understandable for someone coming this far in the code base. it's literally all over the comments & var names, can't avoid it